frame/examples/kitchensink/src/tests.rs

```diff
@@ -89,6 +89,7 @@ impl Config for Test {
 
 // This function basically just builds a genesis storage key/value store according to
 // our desired mockup.
+#[allow(unused)]
```

Comment:
The real Q is: why is there no test in `tests.rs`?

Reply:
Yes, good question.

Reply:
The intent of this example was to demonstrate all frame macros. Since tests do not use any special macro, it only contains the lib and benchmarking code. Happy to add them though if needed.
CC: @kianenigma

Reply:
If adding a single empty test is all it takes, let's just add that.

Reply:
Sounds good, will add one.